README.md

```diff
@@ -1,9 +1,61 @@
 
-### klaviyo-android-sdk
+# klaviyo-android-sdk
 
-# This project is still in pre-alpha
+## This project is still in pre-alpha
 ## Breaking changes are still being made to the API. This is not yet intended for use
 
+## Core SDK
 
+Android SDK allows users to incorporate Klaviyo event and profile tracking functionality within native Android applications.
 
-Android SDK that allows users to incorporate Klaviyo's event and person tracking functionality within native Android applications. Written in Kotlin
+## Push Notifications
+
+### Prerequisites: 
+- Firebase account
+- Familiarity with [Firebase](https://firebase.google.com/docs/cloud-messaging/android/client) client documentation. 
+
+### App Manifest
+Register `KlaviyoPushService` as the service to receive MESSAGING_EVENT intents. 
+This allows the Klaviyo SDK to receive push tokens and foreground notifications. 
+```xml
+<service android:name="com.klaviyo.push.KlaviyoPushService" android:exported="false">
+    <intent-filter>
+        <action android:name="com.google.firebase.MESSAGING_EVENT" />
+    </intent-filter>
+</service>
+```
+[//]: # (TODO we could also make KlaviyoPushService open, and advise sub-classing it) 
```

Comment:
Calling out this for comment:
For devs who want to implement their own messaging service, do y'all think we should we make our `KlaviyoPushService` class `open`, and recommend subclassing it? Or just leave it as I've written it here.

Reply:
If we expand the functionality of the service in the future I could totally see this happening but right now its just two functions with pretty clear use cases so I dont see a need yet

Reply:
Actually I should continue reading the PR before I comment on this, I'm seeing you actually are doing more changes in there than I expected

Reply:
Having read the rest of the PR now, that's also how I'd approach this. There are certain pieces of functionality I don't want them overriding (like our simple push token getter, we expect that in a particular place, I dont want a developer breaking that cause he didnt know what he was doing).

Reply:
Yeah, definitely safer.